m68k-amigaos-gcc10.cmake

```diff
@@ -1,5 +1,6 @@
 set(TOOLCHAIN_OS AmigaOS)
 set(TOOLCHAIN_SYSTEM_INFO_FILE Platform/${TOOLCHAIN_OS})
+cmake_policy(SET CMP0057 NEW)
```

Comment:
why is it needed? to support old cmake version cmakelists?

Reply:
changed behavior in IN_LIST, if CMP0057 is not set to NEW it defaults to the old style, which throws errors in the way we use it.